roles/puddle/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- fail:
+    msg: "Puddle is only relevant on RHEL"
+  when: ansible_distribution != 'RedHat'
+
+# install puddle
+- include: install.yml
+
+# configure NFS
+- include: nfs.yml
+
+# configure puddle
+- include: configure.yml
```

Comment:
Tags might be helpful here if we ever want to re-configure puddle without re-installing it.  Thoughts?